libs/ballot-interpreter/src/legacy_adapter.ts

```diff
@@ -38,23 +41,19 @@ import {
   Rect as NextRect,
   InterpretedContestLayout,
   InterpretedContestOptionLayout,
-  ScoredBubbleMarks,
   ScoredPositionArea,
+  ScoredContestOption,
+  ScoredBubbleMarks,
 } from './hmpb-ts';
-import type {
-  InterpretFileResult,
-  InterpretResult,
-  InterpreterOptions,
-} from './interpret';
+import { type InterpretFileResult, type InterpretResult } from './interpret';
+import { getScoreWriteInsFlag, InterpreterOptions } from './options';
 
 type OkType<T> = T extends Ok<infer U> ? U : never;
 
-function convertNewHampshireNextMarkToSharedMark(
```

Comment:
Glad to see the New Hampshire references removed! It's about time.

I wonder if we should consider renaming the legacy adapter since it's kind of what we've got...